src/index.html

```diff
@@ -6,7 +6,43 @@
     <title>Calendar</title>
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
-  <body>
-    <h1>Calendar</h1>
+  <body class="page">
+    <div class="
+      calendar
+      page__calendar
+      calendar--month-lenght--30
+    ">
+      <div class="calendar__day calendar__day--current-day--1 calendar__day--start-day--sun"></div>
+      <div class="calendar__day calendar__day--current-day--2"></div>
```

Comment:
```suggestion
      <div class="calendar__day"></div>
```

Read the task carefully. You need to add numbers with CSS only.